app/models/shared_supplier.rb

```diff
@@ -18,7 +18,7 @@ def autofill_attributes
   # return list of synchronisation methods available for this supplier
   def shared_sync_methods
     methods = []
-    methods += %w(all_available all_unavailable) if shared_articles.count < 200
+    methods += %w(all_available all_unavailable)  if shared_articles.count < (FoodsoftConfig[:shared_supplier_article_limit] || 200)
```

Comment:
please set it [`get_default_config`](https://github.com/foodcoops/foodsoft/blob/a66a44bdcf124bc108c35a1c5a191ef475d2a6c5/lib/foodsoft_config.rb#L247) instead

Reply:
done